.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--
+  Bonjour!
+
+  We can't express how grateful we are that you're working on making GitPoint
+  better! We're thrilled to take a look at the changes you've made and merge
+  them in as soon as possible. Please fill out this template to make the
+  reviewal process as quick and smooth as possible!
+
+  Thanks again for your hard work!
+-->
+
```

Comment:
We should add more questions/checklist here, to encourage/make sure that:

- Unit tests are provided
- The work was tested on a simulator or device (maybe ask for the OS version too)
- The contributor used `yarn contributors:add` to add him or herself as a contributor




- [ ] Tests provided
- [ ] I have tested this PR

Reply:
For example, as is done in [Symfomy](https://github.com/symfony/symfony/blob/master/.github/PULL_REQUEST_TEMPLATE.md).